Test/CalibrationTests.cs

```diff
@@ -0,0 +1,30 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using TaskLayer;
+
+namespace Test
+{
+    class CalibrationTests
+    {
+        [Test]
+        public static void ExperimentalDesignCalibrationTest()
+        {
+            CalibrationTask calibrationTask = new CalibrationTask();
+            string outputFolder = Path.Combine(TestContext.CurrentContext.TestDirectory, @"TestCalibration");
+            string myFile = Path.Combine(TestContext.CurrentContext.TestDirectory, @"TestData\SmallCalibratible_Yeast.mzML");
+            string myDatabase = Path.Combine(TestContext.CurrentContext.TestDirectory, @"TestData\smalldb.fasta");
+            string filePath = Path.Combine(TestContext.CurrentContext.TestDirectory, @"TestData\ExperimentalDesign.tsv");
+            using (StreamWriter output = new StreamWriter(filePath))
+            {
+                output.WriteLine("FileName\tCondition\tBiorep\tFraction\tTechrep");
+                output.WriteLine("PrunedDbSpectra" + "\t" + "condition" + "\t" + "1" + "\t" + "1" + "\t" + "1");
+            }
+
+            calibrationTask.RunTask(outputFolder, new List<DbForTask> { new DbForTask(myDatabase, false) }, new List<string> { myFile }, "test");
+            File.Delete(filePath);
+        }
```

Comment:
Assert that:
- Experimental Design file exists with the correct calibrated file name
- Calibrated spectra file (mzml) exists
- File-specific toml with the correct name exists and that the suggested ppm tolerances are contained in the toml
- Precursor and product median and interquartile range ppm errors decrease